pkg/providers/ovirt/mapper/mapper.go

```diff
@@ -80,10 +80,23 @@ func NewOvirtMapper(vm *ovirtsdk.Vm, mappings *v2vv1alpha1.OvirtMappings, creds
 }
 
 // CreateEmptyVM creates empty virtual machine definition
-func (o *OvirtMapper) CreateEmptyVM() *kubevirtv1.VirtualMachine {
+func (o *OvirtMapper) CreateEmptyVM(vmName *string) *kubevirtv1.VirtualMachine {
 	return &kubevirtv1.VirtualMachine{
+		ObjectMeta: metav1.ObjectMeta{
+			Labels: map[string]string{
+				"app":                                *vmName,
+				"flavor.template.kubevirt.io/Custom": "true",
```

Comment:
since the labels are repeated in few place, and we already have few of them as constants, let's be consistent for the other labels as well and extract constants for.

Reply:
ack